scripts/ebounty.lic

```diff
@@ -997,6 +1020,29 @@ module EBounty
     
     
     
+    end
+  
+    def self.set_eval
+    
+      bounty_eval = "(checkbounty =~ /^You (?:succeeded|have located)|your task is failed/)" # bounty complete
+      
+      if checkbounty =~ /A local divinist has had visions of the child fleeing/
+        bounty_eval += " || (checkbounty =~ /^You have made contact with the child/ && GameObj.npcs.any? { |npc| npc.name =~ /\\bchild\\b/ })"
+      elsif checkbounty =~ /^You are not currently assigned / && !Spell[9003].active?
+        bounty_eval += " || (checkbounty =~ /^You are not currently assigned / && !Spell[9003].active?)"
+      elsif checkbounty =~ /You have been tasked to retrieve(.*)of at least/
+        EBounty.data.bundle_skin = $2 if checkbounty =~ /You have been tasked to retrieve (\d+) (.*?)(?:s)? of at/        
+        EBounty.data.bundle_skin = EBounty.data.bundle_skin.match(/(\w+)\s+(\w+)\s*\Z/)
+   
+        bounty_eval += ' || (count = 0;GameObj.inv.each{|i| i.contents.each{ |k| if k.name =~ /bundle of #{EBounty.data.bundle_skin}/ && !hidden?; lines = Lich::Util.quiet_command_xml("measure #{k}", /You glance through/);count = $1.to_i if lines.any?{ |l| l=~ /count a total of (\d+) <a exist/};end;};};count >= EBounty.data.remaining_skins.to_i)' 
```

Comment:
I'm pretty sure this needs to be `count = $1.to_i if lines.any?{ |l| l=~ /count a total of (\d+) <a exist/}` Otherwise your count will only be of the last bundle you measured and bounties are often more than 9 skins.